apps/api/src/app/auth/services/auth.service.ts

```diff
@@ -11,7 +11,7 @@ import {
   UserRepository,
 } from '@novu/dal';
 import { AuthProviderEnum, IJwtPayload, ISubscriberJwt, MemberRoleEnum, SignUpOriginEnum } from '@novu/shared';
-import { AnalyticsService } from '@novu/application-generic';
+import { AnalyticsService, CacheKeyPrefixEnum, Cached } from '@novu/application-generic';
```

Comment:
moved caching service and all the interceptors to the `application-generic` package